### DISCUSSION TITLE
Dark mode for launcher please

### DISCUSSION THREAD

**Discussion by JunglePredator** [X2CommunityCore/xcom2-launcher#305] [General]
Hi, thanks for giving us an alternative to the crappy 2k launcher.

Are you able to give the launcher a dark themed skin option?

It's so much easier on the eyes even in a bright room.

**Comment by RevZero** [comment]
Hi. Dark mode was considered at some point, it just doesn't have a high priority. Its also somewhat cumbersome to implement for the AML UI and would require some testing if it is even possible to make it look nice.
I was pretty sure that there was already a related issue filed, but I couldn't find it. If there is none, I can add this as a feature request later.
